apps/api/src/index.ts

```diff
@@ -37,7 +38,7 @@ const createApi = () =>
       {
         isSignedIn: true,
       },
-      (app) => app.use(agentsModule.use(branchesModule)),
+      (app) => app.use(agentsModule.use(branchesModule)).use(providersModule),
```

Comment:
To verify: provider lives under the user / organisation, not under the branch, is that right?